miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultUserService.java

```diff
@@ -72,7 +98,7 @@ public long update(User user) throws IOException {
     }
     if (securityManager.isPasswordMutable()) {
       if (user.getPassword() != null) {
-        original.setPassword(user.getPassword());
+        user.setPassword(validateAndEncodePassword(user.getPassword(), false));
```

Comment:
Should this be `original.setPassword()`? It's `original` that's saved below

Reply:
Yep! fixed